webapp/src/Service/ExternalContestSourceService.php

```diff
@@ -290,6 +303,34 @@ protected function setLastEvent(?string $eventId): void
             ->execute();
     }
 
+    protected function setLastHTTPCode(int $httpCode): void
+    {
+        // We use a direct query since we need to reload the source otherwise and we don't want that since that would
+        // take more queries.
+        $this->em
+            ->createQuery(
+                'UPDATE App\Entity\ExternalContestSource ecs
```

Comment:
I would strongly prefer a sprintf with `ExternalContestSource::class` here so if we ever rename it IDE's will understand.